astrosat_users/models/models_users.py

```diff
@@ -27,6 +27,16 @@ def user_avatar_path(instance, filename):
     return f"users/{instance.username}/{filename}"
 
 
+class UserRegistrationStageType(models.TextChoices):
+
+    USER = "USER", _("User")
+    CUSTOMER = "CUSTOMER", _("Customer")
+    CUSTOMER_USER = "CUSTOMER_USER", _("CustomerUser")
+    ORDER = "ORDER", _("Order")
+
+    __empty__ = _("None")
```

Comment:
Is this a standard way to define `None`. I remember work I did to update model for `Customer` I think, it had an `enum` like this for customer types, the issue wanted a `None` option, but I didn't add one and left it to the frontend as my naive way didn't work. This is just a question for my learning.

Reply:
It's not needed. By default the empty value will be `None`.  This is valid if `null=True` is set on the field definition.  All that this does is to customise the _label_ that is used for form/serializer fields.  And that's not really exposed in our apps.  So I guess there's no real point to this line.  If it bothers you, I can remove it.

Reply:
No, it just reminded me of the change I had made and wondered if this was kinda a standard thing and I had done it a bit wrong. Happy with it, so will resolve this conversation.